packages/suite/src/middlewares/suite/analyticsMiddleware.ts

```diff
@@ -49,7 +50,7 @@ const analyticsMiddleware =
 
         const state = api.getState();
 
-        if (deviceActions.authDevice.match(action)) {
+        if (isAnyOf(authorizeDevice.fulfilled)(action)) {
```

Comment:
nit: `isAnyOf` seems redundant since there is only one action.
```suggestion
        if (authorizeDevice.fulfilled.match(action)) {
```

Reply:
https://github.com/trezor/trezor-suite/pull/12438/commits/545d0ac7a1166da847455312262d1b447369b85b